resources/public/api/conf/1.0/endpoints/create_benefit.yaml

```diff
@@ -0,0 +1,122 @@
+summary: Create State Benefit
+description: |
+  This endpoint allows a developer to create any state benefits they believe are missing.
+  A National Insurance number, tax year, benefit type and start date must be provided.
+
+  This endpoint can only be used for a tax year that has ended.
+
+  ### Test data
+  <p>Scenario simulations using Gov-Test-Scenario headers is only available in the sandbox environment.</p>
+  <table>
+      <thead>
+          <tr>
+              <th>Header Value (Gov-Test-Scenario)</th>
+              <th>Scenario</th>
+          </tr>
+      </thead>
+      <tbody>
+          <tr>
+              <td><p>N/A - DEFAULT</p></td>
+              <td><p>Simulates a list of all state benefit types.</p></td>
+          </tr>
+          <tr>
+             <td><p>BENEFIT_TYPE_EXISTS</p></td>
+             <td><p>Simulates the scenario where a benefit of this type already exists.</p></td>
+          </tr>
+          <tr>
+              <td><p>NOT_FOUND</p></td>
```

Comment:
Don't think this should be here as it is a create endpoint so there is nothing to be found. NOT_FOUND also isn't present in the spec.